tests/extension_tests/deferred_bindings_tests/test_deferred_bindings.py

```diff
@@ -77,9 +77,9 @@ async def test_deferred_bindings_enabled_log(self):
 
 @unittest.skipIf(sys.version_info.minor <= 8, "The base extension"
                                               "is only supported for 3.9+.")
-@unittest.skipIf(sys.version_info.minor < 13, "For python 3.13+,"
-                                              "this logic is in the"
-                                              "library worker.")
+@unittest.skipIf(sys.version_info.minor >= 13, "For python 3.13+,"
```

Comment:
Use a constant to defined the supported python version to minimize the changes in the future.